ZenPacks/zenoss/WSMAN/tests/testWSMANDataSource.py

```diff
@@ -0,0 +1,78 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+from twisted.python.failure import Failure
+
+from Products.ZenEvents import ZenEventClasses
+from Products.ZenTestCase.BaseTestCase import BaseTestCase
+
+from ZenPacks.zenoss.WSMAN.datasources.WSMANDataSource import \
+    WSMANDataSourcePlugin
+from ZenPacks.zenoss.WSMAN.tests.utils import load_pickle
+
+
+class TestWSMANDataSourcePlugin(BaseTestCase):
+
+    def setUp(self):
+        self.plugin = WSMANDataSourcePlugin()
+        self.results = load_pickle('success')
+        self.config = load_pickle('config')
```

Comment:
Usually, you just need to access a couple of attributes from the config and you don't need to keep it in the file, just check the following example out -https://github.com/zenoss/ZenPacks.zenoss.CiscoMonitor/blob/03f8f0786604ff7beac2ace977adbb0490e1eb72/ZenPacks/zenoss/CiscoMonitor/tests/testDatasourcePlugins.py#L23